separatemodels/src/main/java/pl/altkom/asc/lab/cqrs/intro/separatemodels/readmodel/PolicyInfoDtoMapRow.java

```diff
@@ -0,0 +1,39 @@
+package pl.altkom.asc.lab.cqrs.intro.separatemodels.readmodel;
+
+import org.springframework.jdbc.core.RowMapper;
+
+import java.math.BigDecimal;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.time.LocalDate;
+import java.util.UUID;
+
+public class PolicyInfoDtoMapRow implements RowMapper<PolicyInfoDto> {
```

Comment:
I'd change the name to PolicyInfoDtoRowMapper
plus this file has different formatting then in the project, please try to use uniformed formatting